mappings/net/minecraft/world/gen/trunk/LargeOakTrunkPlacer.mapping

```diff
@@ -1,4 +1,9 @@
 CLASS net/minecraft/unmapped/C_wptxwxvu net/minecraft/world/gen/trunk/LargeOakTrunkPlacer
+	FIELD f_eitzscab CLUSTER_DENSITY_MAGIC D
```

Comment:
MAGIC? This one is a bit odd. But idk maybe magic makes sense.

Reply:
Renamed to `CLUSTER_DENSITY_MAGIC_NUMBER`

Reply:
I like that more, since magic number would be random number correct?